moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -166,25 +161,8 @@ struct CollisionResult
     cost_sources.clear();
   }
 
-  /** \brief Throttled warning of the first collision pair, if any. Other collisions are logged at the DEBUG level */
-  void print()
-  {
-    if (!contacts.empty())
-    {
-      ROS_WARN_STREAM_THROTTLE_NAMED(collision_log_period, collision_log_name,
-                                     "Objects in collision (printing first collision pair of "
-                                         << contacts.size() << "): " << contacts.begin()->first.first << ", "
-                                         << contacts.begin()->first.second);
-
-      // Log all collisions at the debug level
-      ROS_DEBUG_STREAM_THROTTLE_NAMED(collision_log_period, collision_log_name, "Objects in collision:");
-      for (auto contact : contacts)
```

Comment:
Here contact entries were copied for no reason. Strange, that clang-tidy didn't flag this issue.

Reply:
thanks, I'll stop making this mistake now